rest-heroes/src/main/kubernetes/minikube.yml

```diff
@@ -51,7 +51,7 @@ data:
   get-data.sh: |-
     #!/bin/bash
     
-    curl https://raw.githubusercontent.com/quarkusio/quarkus-super-heroes/main/rest-heroes/deploy/db-init/initialize-tables.sql --output /docker-entrypoint-initdb.d/1-init-tables.sql
+    curl https://raw.githubusercontent.com/quarkusio/quarkus-super-heroes/rhbq-2.7rest-heroes/deploy/db-init/initialize-tables.sql --output /docker-entrypoint-initdb.d/1-init-tables.sql
```

Comment:
I think you missed a `/` between `rhbq-2.7` and `rest-heroes`

Reply:
What a nice catch. That's what happens when you get overly excited while making your first contribution to a project. Thank you.